src/vg/civcraft/mc/civmodcore/Config.java

```diff
@@ -1,15 +1,13 @@
 package vg.civcraft.mc.civmodcore;
 
-import java.io.IOException;
+import static vg.civcraft.mc.civmodcore.CivModCorePlugin.log;
```

Comment:
Hmm, I might need to think about this for a bit. 

I can't do a static import for ACivMod.log() because that won't use the subclassed plugin's log() method, but ACivMod method. Which will break, since ACivMod doesn't have a logger. 

I changed the Dummy.java to CivModCorePlugin.java so logging CivModCore would work, but I suppose it would make sense to use the subclassed plugin's loggers somehow...